src/commands/MooCommand.ts

```diff
@@ -0,0 +1,37 @@
+import { Message } from 'discord.js';
+import PrefixCommand from './PrefixCommand';
+import { SingleMention } from '../mentions/SingleMention';
+
+export default class PingCommand extends PrefixCommand {

```

Comment:
You forgot to rename this to "MooCommand" here. Not a big deal since it's a default import though.

Reply:
Renamed it to MooCommand